docs/documentation/stories/universal-rendering.md

```diff
@@ -336,6 +336,7 @@ module.exports = {
     new webpack.ContextReplacementPlugin(
       /(.+)?express(\\|\/)(.+)?/,
       path.join(__dirname, 'src'),
+      {}
```

Comment:
Would it not be better to remove the command and third argument altogether, or would this be incorrect?